examples/request/request_object_client_id_did.json

```diff
@@ -7,13 +7,13 @@
   "client_metadata": {
     "vp_formats": {
       "jwt_vp": {
-        "alg": [
+        "alg_values": [
           "EdDSA",
           "ES256K"
         ]
       },
       "ldp_vp": {
-        "proof_type": [
+        "proof_type_values": [
           "Ed25519Signature2018"
```

Comment:
@TimoGlastra could you change this value to `DataIntegrityProof` to resolve this thread https://github.com/openid/OpenID4VP/pull/401#discussion_r2000486728 and issue that Brian opened for it

Reply:
Yes i can update the examples